stories/0-Form.stories.tsx

```diff
@@ -135,6 +135,65 @@ storiesOf("Form", module)
       />
     </FormField>
   ))
+
+  .add("Number Input", () => (
```

Comment:
Let's add an example where `maxFractionDigits` is 0 so that only integers are allows